viewer/core/cad.ts

```diff
@@ -1,5 +1,5 @@
 /*!
- * Copyright 2021 Cognite AS
```

Comment:
I don't think these should be part of this PR - I asked Lars about this last week: https://cognitedata.slack.com/archives/G0115JQ8JM8/p1641199767434200

Reply:
Oh, ok. I think linter did it. Because I haven't touched dates in all these files. Should I reconfigure it or leave as is?

Reply:
This should be fixed :/ I updated the linter to avoid this, I think you simply can rollback commit d827b9100bf6640338d793e932d9285b489bd629

Reply:
Yeah, fixed this one